docs/admin/deploy/scale.mdx

```diff
@@ -124,7 +124,7 @@ A PostgreSQL instance for storing code insights data.
 | `Type`      | Persistent Volumes for Kubernetes                                          |
 |             | Persistent SSD for Docker Compose                                          |
 
-> WARNING: The database must be configured properly to consume resources effectively and efficiently for better performance and to avoid regular usage from overwhelming built-in utilities (like autovacuum for example). For example, a Postgres database running out of memory indicates that it is currently misconfigured and the amount of memory each worker can utilize should be reduced. See our [Postgres database configuration guide](https://docs.sourcegraph.com/admin/config/postgres-conf) for more information.
+> WARNING: The database must be configured properly to consume resources effectively and efficiently for better performance and to avoid regular usage from overwhelming built-in utilities (like autovacuum for example). For example, a Postgres database running out of memory indicates that it is currently misconfigured and the amount of memory each worker can utilize should be reduced. See our [Postgres database configuration guide](https://sourcegraph.com/docs/admin/config/postgres-conf) for more information.
```

Comment:
```suggestion
<Callout type="warning">The database must be configured properly to consume resources effectively and efficiently for better performance and to avoid regular usage from overwhelming built-in utilities (like autovacuum for example). For example, a Postgres database running out of memory indicates that it is currently misconfigured, and the amount of memory each worker can utilize should be reduced. See our [Postgres database configuration guide](/admin/config/postgres-conf) for more information.</Callout>
```